model/src/main/kotlin/licenses/ResolvedLicenseInfo.kt

```diff
@@ -69,16 +69,17 @@ data class ResolvedLicenseInfo(
     /**
      * Return the effective [SpdxExpression] of this [ResolvedLicenseInfo] based on their [licenses] filtered by the
      * [licenseView] and the applied [licenseChoices]. Effective, in this context, refers to an [SpdxExpression] that
-     * can be used as a final license of this [ResolvedLicenseInfo].
+     * can be used as a final license of this [ResolvedLicenseInfo]. [licenseChoices] will be applied in the order they
```

Comment:
Imperative in the commit message:

"Enables to use license choices" -> "Enable to use license choices"